index.md

```diff
@@ -62,6 +62,14 @@ commands, such as `SHOW TABLES` or `SELECT/INSERT/UPDATE/DELETE`.
 
 * [GitHub repository](https://github.com/cloudspannerecosystem/spanner-cli)
 
+### wrench
+
+* [GitHub repository](https://github.com/cloudspannerecosystem/wrench)
+
+wrench is a schema management tool for Cloud Spanner.
+It manages DDLs for schema migration and provides several schema related features
+like create database, delete database and load schema to file.
```

Comment:
Suggesting a minor rewording here:
"like create database, delete database and load schema to file."
-->
"like creating / deleting databases and loading schemas from files."